docs/cloud/details/usage_observability.md

```diff
@@ -1,61 +1,96 @@
 ---
-title: Usage observability
+title: Access and usage insights
 parent: Raito Cloud
 nav_order: 30
 has_children: false
 has_toc: false
 permalink: /docs/cloud/usage_observability
 ---
 
-# Usage observability
+# Access and usage insights
 
 You can add intelligence to managing data access by observing data usage. Raito Cloud displays the data usage from day 1 and continues to do so. These insights are provided on multiple places within Raito Cloud, as they all cover a different angle:
 
-- **Data objects**: all usage of a single data object is grouped on the `Usage` tab of a data object
-- **Users**: all usage from a given user is grouped on the `Usage` tab of a user
-- **Dashboard**: the dashboard provides an aggregated overview of usage, both based on data objects as users.
+- **Data objects**: all usage of a single data object is grouped on the `Insights` tab of a data object
+- **Users**: all usage from a given user is grouped on the `Insights` tab of a user
+- **Access Controls**: all usage from a given access provider is grouped on the `Insights` tab of an access control
+- **Dashboard**: the dashboard provides an aggregated overview of usage, both based on data objects, access controls, as users.
+- **Insights**: the insights page provides insights for users, data objects and access controls.
 
 {: .info }
 > ℹ️ To view observability insights, you need the necessary permissions. See [Raito user management](/docs/cloud/user_management) for more information on which roles can see what in Raito.
 
 
+## Definitions
```

Comment:
[pedantic] Inconsistent use of capitalisation at the start of the definitions [/pedantic]